lib/client-assets.php

```diff
@@ -1338,6 +1338,18 @@ function gutenberg_get_block_categories( $post ) {
 	return apply_filters( 'block_categories', $default_categories, $post );
 }
 
+/**
+ * Loads Gutenberg Locale Data.
+ */
+function gutenberg_load_locale_data() {
+	// Prepare Jed locale data.
+	$locale_data = gutenberg_get_jed_locale_data( 'gutenberg' );
+	wp_add_inline_script(
+		'wp-i18n',
```

Comment:
This adds the locale data. But where do we enqueue `wp-i18n` script for the reusable blocks listing?

Reply:
It's a dependency of `wp-list-reusable-blocks` (It's similar in Gutenberg where `wp-edit-post` is the equivalent of `wp-list-reusable-blocks`)